### ISSUE TITLE
Can not reimplement the mIoU in the paper.

### ISSUE THREAD

**Issue by Zxl19990529** [Luffy03/AGMM-SASS#2]
Hi, thanks for your work. I'm currently following your work. However, I can not reimplement the result in the paper. My reimplementation is just 70.95. The followings are my reimplementation:
1. I use the [this repo](https://github.com/meng-tang/rloss/blob/master/data/pascal_scribble/convertscribbles.m) to generate the scribble masks.
2. I found the resnet101 model in current repo is different from the original one. In original resnet101, the ```conv1``` is a single convolutional layer with kernel size 7*7, stride 2 and output channel 64. In current repo, the ```conv1``` is a "Conv block" with ``` conv(3,64,3,2,1)-bn-relu-conv(64,64,3,1,1)-bn-relu-conv(64,128,3,1,1)```. As I found no pretrained model released, I replace the current one to the pytorch offical res101 model, and load the [pretrained model](https://download.pytorch.org/models/resnet101-63fe2227.pth) from pytorch model zone.

**Comment by Luffy03** [comment]
Thanks for your kind comments.
The pre-trained models and weak labels are released now. Specifically, we use the weak labels provided by TEL.
Feel free to contact me if you have any questions.
